frontend/src/apis/index.js

```diff
@@ -0,0 +1,58 @@
+import { parseTo } from '../utils';
+
+const BASE_PATH = 'http://' + window.location.host;
+
+const myFetch = async (url, config = {}, authToken) => {
```

Comment:
i can change it but i dont see much reason why it couldnt be myFetch. Heck it can be something like `internalFetcher`. `modifiedFetcher`, or `customFetcher`